### PR TITLE
Improved version bumping process using substitutions

### LINKED ISSUE #1236: Add variables for the product version values
Hi team,

After a Google search, I've found that we should create some variables in the `conf.py` file. This way, we no longer need to make insecure and global replace every release.

Here is an example:

```py
'''
Wazuh variables 

Example:
   example_var_name = 'foo'
   rst_epilog = '.. |example_name| replace:: %s' % example_var_name      
                                                                         
In your .rst source:                                                     
   My config value is |example_name|                                     
                                                                         
In your output:                                                              
   My config value is foo
'''
```

In this commit, I've added real examples in https://github.com/wazuh/wazuh-documentation/commit/b0ac9cba1a1dcf28c1d4a803ba6f41ca9233aab7, I've tested it and it seems to be working, the next step is to use it for all version values of our documentation.

### PR DESCRIPTION
**Pull request by davidfmiranda**
Hello team!

This PR closes #1236 

## Description

I have added a function on `config.py` that searches and replaces text. Using this, it will be easier to do version bumps because it will require changing the version in just one place.

Taking into account that there are several different versions depending on the used service, I have defined different variables for Wazuh and Elasticsearch: 

variables

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 

Regards,

David

### CODE REVIEW

File: source/development/packaging/generate-wazuh-kibana-app.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ Execute the ``generate_wazuh_app.sh`` script, with the different options you des
 
   Usage: ./generate_wazuh_app.sh [OPTIONS]
 
-      -b, --branch <branch>     [Required] Select Git branch or tag e.g.v3.11.4-7.4.0
+      -b, --branch <branch>     [Required] Select Git branch or tag e.g.v|WAZUH_LATEST|-|ELASTICSEARCH_LATEST_OVA|
```
Comment (alberpilot) [inline source/development/packaging/generate-wazuh-kibana-app.rst]: Why OVA? It must be Elasticsearch latest.

Comment (alberpilot) [reply source/development/packaging/generate-wazuh-kibana-app.rst]: Same in all this document

File: source/development/packaging/generate-wazuh-splunk-app.rst [hunk 2]
```diff
@@ -32,7 +32,7 @@ Execute the ``generate_wazuh_app.sh`` script, with the different options you des
 
   Usage: ./generate_wazuh_splunk_app.sh [OPTIONS]
 
-      -b, --branch <branch>     [Required] Select Git branch or tag e.g. v3.11.4-7.3.0
+      -b, --branch <branch>     [Required] Select Git branch or tag e.g. v|WAZUH_LATEST|-7.3.0
```
Comment (alberpilot) [inline source/development/packaging/generate-wazuh-splunk-app.rst]: 7.3.0 must be changed by splunk latest which is 8.0.1

File: source/development/packaging/generate-wazuh-kibana-app.rst (hunk 1 above)
Comment (davidfmiranda) [reply source/development/packaging/generate-wazuh-kibana-app.rst]: Done.

File: source/development/packaging/generate-wazuh-splunk-app.rst (hunk 2 above)
Comment (davidfmiranda) [reply source/development/packaging/generate-wazuh-splunk-app.rst]: Fixed.

File: source/conf.py [hunk 3]
```diff
@@ -393,6 +393,30 @@ def minification(actual_path):
...
+
+custom_replacements = {
+    "|WAZUH_LATEST|" : "3.11.4",
+    "|WAZUH_LATEST_ANSIBLE|" : "3.11.4",
+    "|WAZUH_LATEST_KUBERNETES|" : "3.11.4",
+    "|WAZUH_LATEST_PUPPET|" : "3.11.4",
+    "|WAZUH_LATEST_OVA|" : "3.11.4",
+    "|WAZUH_LATEST_DOCKER|" : "3.11.4",
+    "|ELASTICSEARCH_LATEST|" : "7.6.1",
+    "|ELASTICSEARCH_LATEST_OVA|" : "7.6.0",
```
Comment (alberpilot) [inline source/conf.py]: 7.6.1 please

**Pull request merged**